index.html

```diff
@@ -3,21 +3,28 @@
     <head>
         <title>Coding Playlists</title>
         <link rel="stylesheet" href="style.css">
+        <script src="tabs.js"></script>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta name="description" content="Coding playlists to elevate focus, boost your productivity and enhance enjoyment on your coding journey.">
         <meta name="google-site-verification" content="YctCueJjkqNWfppJLoyMYRje4PTP2kzxwuf5C9mp6nI" />
     </head>
     <body>
         <header>
             <h1>Coding Playlists</h1>
-            <h4 id="contribution_link"><a href="https://github.com/gracekishino/coding_playlists" target="_blank">Contribute to Coding Playlists →</a></h4>
+            <button onclick="openPlayer('spotify')">Spotify</button>
+            <button onclick="openPlayer('youtube')">Youtube</button>
+            <button onclick="openPlayer('apple')">Apple</button>
+            <!-- <button onclick="window.location.href='https://github.com/gracekishino/coding_playlists">Contribute</button> -->
+            <!-- <h4 id="contribution_link"><a href="https://github.com/gracekishino/coding_playlists" target="_blank">Contribute to Coding Playlists →</a></h4> -->
```

Comment:
Just removing commented out code. Let's put the contribution link in the footer underneath the text in the footer:

`<a  id="contribution_link" href="https://github.com/gracekishino/coding_playlists" target="_blank">Contribute to Coding Playlists →</a>`

```suggestion

```